nixos/lib/testing/driver.nix

```diff
@@ -123,6 +124,18 @@ in
       defaultText = "hostPkgs.qemu_test";
     };
 
+    globalTimeout = mkOption {
+      description = mdDoc ''
+        A global timeout for the complete test, expressed in seconds.
+        Beyond that timeout, every resource will be killed and released and the test will fail.
+
+        By default, we use a 24 hours timeout.
```

Comment:
24 sounds very long. What about 6 hours? Do we have any tests that run longer? If they do it's probably better if they manually override the timeout anyway.